windows/winmm.mpl

```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 2021 Matway Burkow
+#
+# This repository and all its contents belong to Matway Burkow (referred here and below as "the owner").
+# The content is for demonstration purposes only.
+# It is forbidden to use the content or any part of it for any purpose without explicit permission from the owner.
+# By contributing to the repository, contributors acknowledge that ownership of their work transfers to the owner.
+
+"control.Natx" use
+"winmmPrivate" use
+
+winmmInternal: {
+  JOYINFOEX: @JOYINFOEX;
+
+  getJoyPosEx: @joyGetPosEx;
```

Comment:
There should be comment, like this https://github.com/Matway/mpl-sl/blob/e53da6eb03aad8a4021bdc870c4c0a613fad9923/windows/kernel32.mpl#L59 one.

Reply:
fixed